scripts/run_linters.py

```diff
@@ -0,0 +1,95 @@
+"""
+Lints the code for various components. Run with `python3 scripts/run_linters.py` from the root directory of the aqueduct repo.
+
+Requirements:
+- For the Python code linting, please install the following:
+`pip3 install --upgrade black mypy pydantic types-croniter types-requests types-PyYAML`
+- For the Golang linter, please install `golangci-lint` following the instruction here: https://golangci-lint.run/usage/install/
+- For the UI linter, please install node with the version suggested by running `nvm use` from `src/ui`.
+
+If you don't specify any component flag, the script will lint all components. Keep in mind that UI
+takes longer to lint.
+"""
+
+import argparse
+import os
+import subprocess
+import sys
+
+from os.path import join
+
+
+def execute_command(args, cwd=None):
+    with subprocess.Popen(args, stdout=sys.stdout, stderr=sys.stderr, cwd=cwd) as proc:
+        proc.communicate()
+        if proc.returncode != 0:
+            raise Exception("Error executing command: %s" % args)
+
+def lint_python(cwd):
+    execute_command(["black", join(cwd, "src/python"), "--line-length=100"])
```

Comment:
This is the type of stuff we should get out of the habit of to be better about supporting Windows in the future. Rather than doing `join(cwd, 'src/python')`, we can just do `join(cwd, 'src', 'python')`